.github/workflows/loadtest.yaml

```diff
@@ -0,0 +1,45 @@
+name: Performance Test
+on: 
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning'
+      tags:
+        description: 'K6 Load Test'  
+
+jobs:
+  k6_test_run:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          repository: longwuyuan/ingress-nginx
+          ref: k6
+
+      - name: Make dev-env
+        run: |
+          mkdir $HOME/.kube
+          make dev-env
+          sleep 60
+          kubectl get all -A
+
+      - name: Create workload using nginx-alpine
+        run: |
+          kubectl create deploy k6 --image nginx:alpine --port 80 && kubectl expose deploy k6 --port 80 && kubectl create ing k6 --class nginx --rule localhost/=k6:80
+          sleep 15
```

Comment:
Same, use kubectl wait for the Pod here!

Reply:
Changed to `kubectl wait`. Please check @rikatz